src/openforms/registrations/contrib/zgw_apis/api/urls.py

```diff
@@ -12,10 +12,6 @@
 urlpatterns = [
     path("catalogues", CatalogueListView.as_view(), name="catalogue-list"),
     path("case-types", CaseTypesListView.as_view(), name="case-type-list"),
+    path("document-types", DocumentTypesListView.as_view(), name="iotypen-list"),
```

Comment:
Could we also change `iotypen-list` to `document-type-list`?

Reply:
Yeah I think now would be a good time, should be a simple find and replace. I'll get it updated after refinement.